### PR TITLE
12 - Details and delete

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# Summary
- I implemented the _Update()_ function in the _Repository.cs_
- I added the Details and Delete options.
- I implemented a confirmation view for Delete option.

# Trello Card
https://trello.com/c/6VTpumgS

# Summary
image
image
image

### CODE REVIEW

File: training-net/src/training-net.csproj [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.1"/>
     <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.1.1.1"/>
     <PackageReference Include="LocalizationCultureCore" Version="1.1.2"/>
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="3.0.0"/>
   </ItemGroup>
-</Project>
+</Project>
```
Comment (truccomarcos) [inline training-net/src/training-net.csproj]: Add an empty line here :smile: this file tends to delete that empty line when adding a new package

File: training-net/src/Resources/en-US.json [hunk 2]
```diff
@@ -1,13 +1,17 @@
 {
```
Comment (truccomarcos) [inline training-net/src/Resources/en-US.json]: Some of the values on this json are set on Spanish for some reason. Please change them to the proper language :slightly_smiling_face:

File: training-net/src/Controllers/MovieController.cs [hunk 3]
```diff
@@ -101,12 +101,56 @@ public IActionResult Edit(int id, [FromForm] MovieViewModel movieVM)
...
             {
                 throw;
             }
         }
+
+        [HttpGet("Details")]
+        public IActionResult Details(int? id)
+        {
+            if (id == null)
+                return NotFound();
```
Comment (truccomarcos) [inline training-net/src/Controllers/MovieController.cs]: Don't forget to change this validations to use a try-catch block instead

File: training-net/src/Controllers/MovieController.cs [hunk 4]
```diff
@@ -101,12 +101,56 @@ public IActionResult Edit(int id, [FromForm] MovieViewModel movieVM)
...
+            movieVM.ReleaseDate = movie.ReleaseDate;
+            movieVM.Title = movie.Title;
+            return View(movieVM);
+        }
+        
+        [HttpGet("Delete")]
+        public IActionResult Delete(int? id)
+        {
+            if (id == null)
+                return NotFound();
```
Comment (truccomarcos) [inline training-net/src/Controllers/MovieController.cs]: Same here

File: training-net/src/training-net.csproj (hunk 1 above)
Comment (mar7in91Wolox) [reply training-net/src/training-net.csproj]: Done!

File: training-net/src/Resources/en-US.json (hunk 2 above)
Comment (mar7in91Wolox) [reply training-net/src/Resources/en-US.json]: Ups!

File: training-net/src/Controllers/MovieController.cs (hunk 3 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/MovieController.cs]: Done!

File: training-net/src/Controllers/MovieController.cs (hunk 4 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/MovieController.cs]: Done!

File: training-net/src/Controllers/MovieController.cs (hunk 3 above)
Comment (mar7in91Wolox) [reply training-net/src/Controllers/MovieController.cs]: Done!

**Pull request merged**